### PR TITLE
Added italian cuss words.

### PR DESCRIPTION
**Pull request by s-dimaria**

### CODE REVIEW

File: it.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const cuss = {
   'bel bocconcino': 1,
   belini: 2,
   belino: 2,
+  dio: 2,
```
Comment (wooorm) [inline it.js:65]: I don’t think `god` is always a slur. https://github.com/words/cuss/blob/e09ba41d2b2398057a5c5cbaaaf235ae51ba468d/index.js#LL674C3-L674C6

Comment (s-dimaria) [reply it.js]: I edit the fork, i added new words and I reduce "dio" word to 1.

File: it.js [hunk 2]
```diff
@@ -581,6 +585,7 @@ export const cuss = {
   more: 0,
   mori: 0,
   moro: 0,
+  muori: 2,
```
Comment (wooorm) [inline it.js:588]: And this is `die` right? That seems like it might also sometimes be used in normal, clean text?

**Review by wooorm — COMMENTED** [review]
One more Q!

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks, released! https://github.com/words/cuss/releases/tag/2.2.0

**Pull request merged**